pylsp/plugins/references.py

```diff
@@ -8,20 +8,19 @@
 
 
 @hookimpl
-def pylsp_references(document, workspace, position, exclude_declaration=False):
-    with workspace.report_progress("references"):
-        code_position = _utils.position_to_jedi_linecolumn(document, position)
-        usages = document.jedi_script().get_references(**code_position)
+def pylsp_references(document, position, exclude_declaration=False):
+    code_position = _utils.position_to_jedi_linecolumn(document, position)
+    usages = document.jedi_script().get_references(**code_position)
```

Comment:
Also here? This one is explicitly triggered by the user.

Reply:
see above